src/@batch-flask/extensions/angular/forms.ts

```diff
@@ -1,11 +1,11 @@
-import { AsyncValidatorFn, ValidatorFn } from "@angular/forms";
-import { Observable } from "rxjs";
-
 /**
  * Add some typing to the angular forms
  */
 declare module "@angular/forms/src/model" {
 
+    import { AbstractControlOptions, AsyncValidatorFn, ValidatorFn } from "@angular/forms";
```

Comment:
don't think that's show you should have fixed that one. If it doesn't let you provide a type on the FormControl that means this extension class is not working

Reply:
What its doing here is just defining a new module which never gets imported Check where `@angular/forms/src/model` moved to

Reply:
Changed to @angular/forms/forms and fixed this issue